packages/date-picker/src/DatePicker/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { Dispatch, SetStateAction, useEffect, useRef } from "react"
+import DayPicker from "react-day-picker/DayPicker"
+import cx from "classnames"
+import { NavbarElementProps } from "react-day-picker/types/Props"
+import { Modifier, DayModifiers } from "react-day-picker/types/Modifiers"
+import { CalendarNav } from "../CalendarNav/CalendarNav"
+import { defaultCalendarClasses } from "./CalendarClasses"
+import calendarStyles from "./Calendar.scss"
+
+export type CalendarProps = {
```

Comment:
Might want to extend native div props here, and use `style` instead of a custom `styles` prop?

Reply:
I went down this road when I was originally trying to type everything after moving the calendar into its own file. The problem is this isn't a native style, it's created by Popper in the parent and passed down, so I needed to type it accordingly.